lib/core/src/text.rs

```diff
@@ -175,22 +213,15 @@ impl<'a,'b,Str:AsRef<str>> TextComponentBuilder<'a,'b,Str> {
         let gl_context        = self.workspace.context.clone();
         let gl_program        = self.create_program(&gl_context);
         let gl_msdf_texture   = self.create_msdf_texture(&gl_context);
-        let lines             = self.split_lines();
-        let font              = self.fonts.get_render_info(self.font_id);
         let display_size      = self.size / self.text_size;
-        let content_ref       = ContentRef{lines:lines.as_ref(),font};
-        let buffers           = TextComponentBuffers::new(&gl_context,display_size,content_ref);
+        let mut content       = TextComponentContent::new(self.font_id,self.text.as_ref());
+        let buffers           = TextComponentBuffers::new(&gl_context,display_size,content.refresh_info(self.fonts));
```

Comment:
I don't like the name of the variable. What's about `i_am_a_way_too_long_line` instead? :D